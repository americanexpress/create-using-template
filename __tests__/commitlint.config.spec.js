const config = require('../commitlint.config');

describe('commitlint config', () => {
  it('should match ths snapshot', () => {
    expect(config).toMatchSnapshot();
  });
});
