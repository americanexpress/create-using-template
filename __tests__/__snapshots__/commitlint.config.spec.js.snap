// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`commitlint config should match ths snapshot 1`] = `
Object {
  "extends": Array [
    "@commitlint/config-conventional",
  ],
  "rules": Object {
    "scope-case": Array [
      2,
      "always",
      Array [
        "pascal-case",
        "camel-case",
        "kebab-case",
      ],
    ],
  },
}
`;
